elastic/kibana
# [Vislib] Use timestamp on brush event instead of iso dates

## Summary

Closes #90923.
Url drilldowns on brush event for all visualizations emit timestamps instead of the vislib xy axis that emit ISO strings. We want to change this behavior and emit timestamps instead in order to be in line with the rest. 

Url drilldowns are in beta and introduced on 7.10 so we are not going to proceed with migrating the existing URL templates.

### How to test it
Create a URL drilldown (range) and add the following URL template
```{{kibanaUrl}}/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'{{date event.from}}',to:'{{date event.to}}'))&_a=(columns:!(_source),filters:{{rison context.panel.filters}},index:'{{context.panel.indexPatternId}}',interval:auto,query:(language:{{context.panel.query.language}},query:'{{context.panel.query.query}}'),sort:!())```

It should work correctly now with the `date` function
